tee-worker/docker/lit-di-evm-identity-test.yml

```diff
@@ -17,7 +17,10 @@ services:
                 condition: service_healthy
         networks:
             - litentry-test-network
-        entrypoint: "bash -c '/usr/local/worker-cli/lit_ts_test.sh test-di-evm-identity 2>&1' "
+        entrypoint: |
+            /usr/local/worker-cli/lit_ts_api_package_build.sh -p 9912 -u ws://litentry-node -W http://litentry-node -V wss://litentry-worker-1 -A 2011 -C /usr/local/bin/litentry-cli 2>&1
+            && \
+            /usr/local/worker-cli/lit_ts_test.sh test-resuming-worker 2>&1
```

Comment:
Looks like all copy-paste error within these xxx-test.yml files. 
Besides, it's better to keep `bash -C` from the old file.